terraform/modules/len/lambdas/len.py

```diff
@@ -90,6 +76,83 @@ def post_event(auth_token: str):
             "Content-Length": len(event_request_data)
         }
     )
-    logger.info(f"## Posting death certificate {death_certificate}")
+    logger.info(f"## Posting death certificate {death_registration_id}")
     request.urlopen(event_request).read()
-    logger.info(f"## Successfully posted death certificate {death_certificate}")
+    logger.info(f"## Successfully posted death certificate {death_registration_id}")
+
+
+def create_death_registration():
+    connection = psycopg2.connect(user=lev_username, password=lev_password, database=lev_db_name, host=lev_host)
+    cursor = connection.cursor()
+    cursor.execute("select nextval('death_registration_ids')")
+    death_registration_id = cursor.fetchone()
+
+    cursor.execute("""
+        INSERT INTO death_registration_v1 (id, data, forenames, surname, date_of_birth, date_of_death)
+        VALUES(%(id)s, %(data)s, %(forenames)s, %(surname)s, %(date_of_birth)s, %(date_of_death)s)
+        """,
+                   generate_death_registration(death_registration_id))
```

Comment:
indentation seems a bit weird here but it's what intellij told me to do - I don't know python well enough to know if this is right

Reply:
It's aligning on the open bracket. If you move the opening `"""` down a line it should look more normal

Reply:
Nice, thanks!